utils/api.py

```diff
@@ -79,21 +79,22 @@ def tasks_create_file():
     platform = request.forms.get("platform", "")
     tags = request.forms.get("tags", None)
     custom = request.forms.get("custom", "")
-    memory = request.forms.get("memory", False)
+    memory = request.forms.get("memory", 'False')
     clock = request.forms.get("clock", None)
     shrike_url = request.forms.get("shrike_url", None)
     shrike_msg = request.forms.get("shrike_msg", None)
     shrike_sid = request.forms.get("shrike_sid", None)
     shrike_refer = request.forms.get("shrike_refer", None)
-    
-    if memory.isdigit() and int(memory):
-        memory = True
-    elif memory:
+
```

Comment:
dist.py is breaking submit. This should keep the same behavior.  The types were being used inconsistently and I standardized them as a string, then moved them to True/False the same way you originally had in your code.